### PR TITLE
Create derstandard.at.xml

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: src/chrome/content/rules/derstandard.at.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	<test url="http://mobil.derstandard.at/img/ui/searchLupe.gif" />
+	<test url="http://www.derstandard.at/img/ui/searchLupe.gif" />
+	<test url="http://derstandard.at/js/fixedfixed.js" />
+	<test url="http://mobil.derstandard.at/js/fixedfixed.js" />
+	<test url="http://www.derstandard.at/js/fixedfixed.js" />
+	<test url="http://derstandard.at/wetter/Content/img/search-icon2.png" />
+	<test url="http://mobil.derstandard.at/wetter/Content/img/search-icon2.png" />
+	<test url="http://www.derstandard.at/wetter/Content/img/search-icon2.png" />
+
+	<rule from="^http://(www\.)?derstandard\.at/(ajax|css|img|js|wetter/Content/img)/"
```
Comment (TheNavigat) [inline src/chrome/content/rules/derstandard.at.xml]: That line causes many 404s. The page loads without any CSS.

File: src/chrome/content/rules/derstandard.at.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		to="https://derstandard.at/$1/"/>
+	<rule from="^http://mobil\.derstandard\.at/(ajax|css|img|js)/"
+		to="https://mobil.derstandard.at/$1/"/>
```
Comment (TheNavigat) [inline src/chrome/content/rules/derstandard.at.xml]: The dot here is not escaped correctly.

File: src/chrome/content/rules/derstandard.at.xml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	<rule from="^http://epaper.derstandard.at/"
```
Comment (TheNavigat) [inline src/chrome/content/rules/derstandard.at.xml]: The dot here is not escaped correctly. Same applies to the following rule.

Comment (J0WI) [inline src/chrome/content/rules/derstandard.at.xml]: Thanks for the feedback. The test server was down today, so I wasn't able to continue.

**Comment by J0WI** [comment]
Now it seems to work and passed all tests.

File: src/chrome/content/rules/derstandard.at.xml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+<ruleset name="derStandard.at (partial)">
+	<target host="derstandard.at"/>
+	<target host="www.derstandard.at"/>
+	<target host="epaper.derstandard.at"/>
+	<target host="images.derstandard.at"/>
+	<target host="mobil.derstandard.at"/>
+
+	<exclusion pattern="^http://derstandard\.at/$" />
+	<exclusion pattern="^http://www\.derstandard\.at/$" />
+	<exclusion pattern="^http://mobil\.derstandard\.at/$" />
```
Comment (TheNavigat) [inline src/chrome/content/rules/derstandard.at.xml]: I think those are useless since the following rules don't include those links anyway.

Comment (J0WI) [inline src/chrome/content/rules/derstandard.at.xml]: These are necessary to pass the tests.

Comment (TheNavigat) [inline src/chrome/content/rules/derstandard.at.xml]: Oh, I see... That doesn't look like a good way to solve that problem, but I'm not sure if there's a better way to do it without restructuring the whole rewrite system.

So yea, LGTM.

**Comment by J0WI** [comment]
*ping* @fuglede

**Comment by numismatika** [comment]
I would add 
```
	<rule from="^http://livestat\.derstandard\.at/"
		to="https://livestat.derstandard.at/" />	
```
as well.

**Comment by J0WI** [comment]
Thanks!

This site is on Alexa rank 12 in Austria, hopefully someone has time to review this :)
@fuglede @reedy

**Comment by semenko** [comment]
LGTM, :fireworks: Thanks! :fireworks:

**Pull request merged**